tests/unit/test_create_movie.py

```diff
@@ -1 +1,10 @@
 # TODO: Feature 2
+from src.models.movie import Movie
+
+def test_create_movie():
+    movie = Movie('Forrest Gump', 'Robert Zemeckis', 4)
+
+    assert type(movie) == Movie
+    assert movie.title == 'Forrest Gump'
+    assert movie.director == 'Robert Zemeckis'
+    assert movie.rating == 4
```

Comment:
The method here you should be testing is from the `create_movie` method of the movie repository. So you should import the movie repository singleton here and use that to create a movie and make sure it exists.